intl.config.ts

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
The I8n need to go in the specific module

`modules/auth/supabase-auth/lang` 

Each module will have separate and independent translations.
And some how need to be imported here (first hardcoded after dynamically) and merged.
Maybe with the prefix of the module `supabase-auth.Login`